ergo/platforms/predictit.py

```diff
@@ -0,0 +1,332 @@
+"""
+This module lets you get question and prediction information from PredictIt
+and submit predictions, via the API (https://predictit.freshdesk.com/support/solutions/articles/12000001878)
+
+**Example**
+In this example, we predict the odds of democrats sweeping the house, senate, and presidency in the 2020 election based on the combined predictit odds.
```

Comment:
This example is cool but I think it's going to break if either Metaculus or PredictIt questions change/close. Move it into a notebook under `notebooks/` and only add a doctest that we're pretty sure won't break in the next year or two?